src/Tribe/Tickets.php

```diff
@@ -1892,12 +1892,13 @@ public static function add_frontend_stock_data( array $tickets ) {
 
 			tribe_asset(
 				$plugin,
-				'tribe_tickets_frontend_tickets',
+				'tribe-tickets-block',
 				'tickets-block.js',
-				[ 'jquery', 'jquery-ui-datepicker', 'wp-util-not-in-footer', 'wp-i18n' ],
+				[ 'jquery', 'tribe-common', 'jquery-ui-datepicker', 'wp-util-not-in-footer', 'wp-i18n' ],
 				null,
 				[
 					'type'         => 'js',
+					'groups'       => [ 'tribe-tickets-block-assets' ],
```

Comment:
:warning: Array double arrow not aligned correctly; expected 3 space(s) between "'groups'" and double arrow, but found 7.

<sub>hash: 0b91b65253c432a7bd0d55e7ea4a8697</sub>